index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Roboto"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Poppins"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Inter"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Crete"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=R"
+   />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- first  section -->
+    <div class="wrapper">
+      <!-- navbar  section -->
+      <nav class="navbar-section">
+          <span class="nav-span">Victor David</span>
+              <span><a href="#"><img src="./Images/Union.png" alt=""></a></span>
+      </nav>
+      <!-- header  section -->
+      <header>
+          <div class="header-text">
+              <h1>
+                Hey There.
+                I’m Victor
+              </h1>
+          </div>
+          <div class="header-smaller-text">
+              <h3>   I am a Software Developer </h3>
+          </div>
+          <div class="header-paragraph">
+              <p>
+                  I can help you build a product ,
+                  feature or website Look through some of my work and experience!
+                   If you like what you see and have a project you need coded, 
+                  don’t hestiate to contact me.
+               </p>
+
+          </div>
+      </header>
+      <!-- Icons section -->
+      <div class="header-icons">
+        
+          <span><a href="#"><img src="./Images/Github.png" alt=""></a></span>
+          <span><a href="#"><img src="./Images/Icon-Linkedin.png" alt=""></a></span>
+          <span><a href="#"><img src="./Images/Vector.png" alt=""></a></span>
+          <span><a href="#"><img src="./Images/medium.png" alt=""></a></span>
+          <span><a href="#"><img src="./Images/Twitter.png" alt=""></a></span>
```

Comment:
- [ ] Icons are looking great! Let's use `ul` and `li` tags to list them, this is a better way to list items in HTML.